cmd/tsbs_generate_data/main.go

```diff
@@ -48,6 +48,7 @@ const (
 	errInvalidFormatFmt = "invalid format specifier: %v (valid choices: %v)"
 
 	inputBufSize = 4 << 20
+	defaultWriteSize  = 4 << 20 // 4 MB
```

Comment:
these seem to be duplicates, I think you can remove my `inputBufSize` since your name is actually more accurate

Reply:
Ok, thanks - removed